modules/sr/robot/ruggeduino.py

```diff
@@ -5,6 +5,10 @@
 
 
 def init_ruggeduino_array(webot: Robot) -> 'List[Ruggeduino]':
+    """
+    The names in these arrays correspond to the names given to devices in Webots
+    """
+
```

Comment:
Suggest that this be a comment rather than a docstring -- the detail doesn't matter to callers, nor describes how to use this function.